tests/director/tests/application/application_api_test.go

```diff
@@ -2025,6 +2025,127 @@ func TestGetApplicationsAPIEventDefinitions(t *testing.T) {
 	assert.Equal(t, app.EventDefinition.ID, event.ID)
 }
 
+func TestListApplicationsByLocalTenantID(t *testing.T) {
+	//GIVEN
+	ctx := context.TODO()
+	tenantId := tenant.TestTenants.GetDefaultSubaccountTenantID()
+
+	localTenantID := "local-tenant-id-1234"
+
+	nameField := "name"
+	displayNameField := "displayName"
+	localTenantIDField := "localTenantId"
+
+	createAppTemplate := func(name string) graphql.ApplicationTemplate {
+		input := fixtures.FixAppTemplateInputWithDefaultDistinguishLabel(name, conf.SubscriptionConfig.SelfRegDistinguishLabelKey, name+" "+conf.SubscriptionConfig.SelfRegDistinguishLabelValue)
```

Comment:
I think there is no need to use `FixAppTemplateInputWithDefaultDistinguishLabel`, because this "distinguish label" is responsible for representing SaaS Applications and is used for the subscription flow e2e tests.

Reply:
I also prefer to have only what's needed for the test to run. I first tried it with another function (I think it was this one https://github.com/kyma-incubator/compass/blob/94decb80bf142d5affc60604979194644cbfd873/tests/pkg/fixtures/application_template_requests.go#L14), but got an error (either distinguish label had to be provided or something else) and then I switched to this as the other tests in this file also use it.